### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.body-footer): A11y_code.visualstudio.com_AI4W

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-docs#7275]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**
## GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #Chrome; #WCAG1.4.3; #WCP; #Contrast(minimum); #Visual Studio Code Services; #A11yAuto; #AI4W; #BM_VisualStudioCodeServices_Web_Apr2023; #Win11;#color-contrast;

## Environment Details: 
Application Name:  code.visualstudio.com
URL: https://code.visualstudio.com/docs/java/java-project
Microsoft Edge version 125.0.0.0 (Official Build) (64-bit)

## Repro Steps:
1. Hit the URL: https://code.visualstudio.com/docs/java/java-project
2. Run fast pass and observe the issue.

## Issue: 
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

## Target application: 
Java project management in Visual Studio Code - https://code.visualstudio.com/docs/java/java-project

## Element path: 
.body-footer

## Snippet: 
<div class="body-footer">4/25/2022</div>

## Related paths: 
  #theme-v2

### Fix any of the following:
  Element has insufficient color contrast of 3.89 (foreground color: #707070, background color: #0e0e0e, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

## Environment: 
Microsoft Edge version 125.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

## Attachment:
![image](https://github.com/microsoft/vscode-docs/assets/100269580/1d13dff9-0e36-40f8-9463-e76ffd5c901c)

**Comment by daviddossett** [comment]
Weird, I could've sworn I already fixed this. Fixed in [vscode-website#1760](https://github.com/microsoft/vscode-website/pull/1760), in any case.
